Patch-Source: https://github.com/emacs-circe/circe/commit/0bb4ee83693ef3a3504e3cdc512b218afb333af3.patch
--
From 0bb4ee83693ef3a3504e3cdc512b218afb333af3 Mon Sep 17 00:00:00 2001
From: Ola Nilsson <ola.nilsson@gmail.com>
Date: Tue, 5 Mar 2024 22:34:26 +0100
Subject: [PATCH] Add lexical-cast to test-tracking.el

Buttercup 1.34 requires lexical-cast: t in all files defining
buttercup test suites for Emacs >= 29.
It was already documented as a requirement, but was only actually
required for some specific functionality.  The introduction of
Oclosures for Emacs versions that support them made it a hard
requirement.
---
 tests/test-tracking.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-tracking.el b/tests/test-tracking.el
index 35076917..21b252a7 100644
--- a/tests/test-tracking.el
+++ b/tests/test-tracking.el
@@ -1,4 +1,4 @@
-;;; Automated tests for tracking.el
+;;; test-tracking.el --- Automated tests for tracking.el -*- lexical-binding: t; -*-
 
 (require 'tracking)
 
